lib/internal/Magento/Framework/App/Response/HttpInterface.php

```diff
@@ -16,4 +16,49 @@
      * @return void
      */
     public function setHttpResponseCode($code);
+
+    /**
+     * Set a header
+     *
+     * If $replace is true, replaces any headers already defined with that
```

Comment:
1. Please add/fix description for all new methods in the interface. E.g. here it does not make sense to split lines 23 and 24
 2. Please change description of these methods in implementing classes to `{@inheritdoc}`